### DISCUSSION TITLE
Check-in

### DISCUSSION THREAD

**Discussion by objecttothis** [opensourcepos/opensourcepos#3681] [Team Posts]
Sorry, I've been silent.  War has broken out in the country I'm in and it has occupied much of my attention.  Once we are ready to begin reviewing my code changes I will make myself available to make whatever changes are requested and answer questions. 

I do want to say, don't not begin reviewing my code changes because of the situation on the ground here.  I still really need to merge those changes.

**Comment by jekkos** [comment]
HI @objecttothis. Indeed I saw some reports on the news on this. I hope you and your family are safe and in good health. Let's hope they come to a ceasefire agreement soon.

I guess we might need to handle some bug reports before I release 3.3.3. But then after I'll have a look at your work there indeed. 
Today I came across an interesting read on [PHP 8.](https://stitcher.io/blog/php-8-before-and-after) It seems to support a kind of subscribe mechanism natively now, looked quite nice. 

Also personally I have upgraded my install to 7.4 recently and I must say it was worth the effort. The pos feels a lot snappier now.

**Comment by objecttothis** [comment]
Thanks @jekkos. We aren't close to the fighting, but government censoring of the internet has caused headaches.  Everyone is healthy and that's the main thing.

No worries on sorting out issues with 3.3.3.

Interesting note about PHP 8.  I've been running 7.4.x variants for several months and it definitely is a step up.  My optimizations should make an improvement as well.
